addons/mail/static/src/scss/variables/primary_variables.scss

```diff
@@ -18,3 +18,5 @@ $o-mail-composer-text-input-height: 40px !default;
 $o-mail-LinkPreview-width: 320px !default;
 $o-mail-LinkPreview-height: 240px !default;
 $o-mail-LinkPreviewCard-height: 80px !default;
+
+$o-mail-ActivityListPopoverItem-assignee-avatar-size: 1.5rem !default;
```

Comment:
No need for this variable, it's a very specific size, so can just put 1.5rem immediately where needed